gen/definitions/device_vni_interface.yaml

```diff
@@ -0,0 +1,225 @@
+---
+name: Device VNI Interface
+rest_endpoint: /api/fmc_config/v1/domain/{DOMAIN_UUID}/devices/devicerecords/%v/vniinterfaces
+doc_category: Devices
+test_tags: [TF_VAR_device_id]
+attributes:
+  - tf_name: device_id
+    type: String
+    reference: true
+    requires_replace: true
+    description: UUID of the parent device (fmc_device.example.id).
+    example: 76d24097-41c4-4558-a4d0-a8c07ac08470
+    test_value: fmc_device_physical_interface.test.device_id
+  - model_name: name
```

Comment:
With FTD being the only supported device atm (I believe) we could construct it from the `vniId` for now and if we later need to make it configurable we could introduce an optional attribute to overwrite it. What do you think?

Reply:
done, thanks